templates/frameworks/list.html.twig

```diff
@@ -113,98 +111,45 @@
-                        {#View all#}
-                        {#</label>#}
-                        {#</div>#}
-                        {#</div>#}
-                        {#</fieldset>#}
-                        {#</div>#}
-
-
                     </form>
 
                 </div>
-                <div class="govuk-grid-column-two-thirds">
-
-                    {% if pagination.totalResults == 1 %}
-                        <h2 class="govuk-heading-m ccs-font-weight-semibold govuk-!-font-size-24">{{ pagination.totalResults }}
-                            framework found</h2>
+                {% if guided_match_flag is defined %}
+                    {% if query is defined and query in keywords%}
+                    <div class="govuk-grid-column-two-thirds guided-match-background govuk-!-margin-bottom-2 guided-match-area">
+                        {% if pagination.totalResults == 1 %}
+                            <h2 class="govuk-heading-m ccs-font-weight-semibold govuk-!-font-size-24">{{ pagination.totalResults }} framework found</h2>
+                        {% else %}
+                            <h2 class="govuk-heading-l ccs-font-weight-semibold text-white govuk-!-margin-1">Get help finding the right framework</h2>
+                            <h2 class="govuk-heading-s ccs-font-weight-semibold text-white">There are "{{ pagination.totalResults }}" results found for {{ query }}. Depending on your requirements, some of these frameworks maybe better suited than others.</h2>
+                        {% endif %}
+
+                        <a href={{ match_url }} role="button" draggable="false" class="govuk-button govuk-button--start govuk-!-margin-bottom-3" data-module="govuk-button">
+                            Start now
+                            <svg class="govuk-button__start-icon" xmlns="http://www.w3.org/2000/svg" width="17.5" height="19" viewBox="0 0 33 40" aria-hidden="true" focusable="false">
+                            <path fill="currentColor" d="M0 0h13l20 20-20 20H0l20-20z" />
+                            </svg>
+                        </a>
+                    </div>
                     {% else %}
-                        <h2 class="govuk-heading-m ccs-font-weight-semibold govuk-!-font-size-24">{{ pagination.totalResults }}
-                            frameworks found</h2>
+                    <div class="govuk-grid-column-two-thirds">
+
+                        {% if pagination.totalResults == 1 %}
+                            <h2 class="govuk-heading-m ccs-font-weight-semibold govuk-!-font-size-24">{{ pagination.totalResults }}
+                                framework found</h2>
+                        {% else %}
+                            <h2 class="govuk-heading-m ccs-font-weight-semibold govuk-!-font-size-24">{{ pagination.totalResults }}
+                                frameworks found</h2>
+                        {% endif %}
+                    
+                    </div>
                     {% endif %}
```

Comment:
This else path is against `{% if query is defined and query in keywords%}`
but that does not need an 'else'

Reply:
If it depends on the flag it needs be there. If the flag is not important nor controls the conditional flow then it needs to be there because they lead to different paths.